### DISCUSSION TITLE
请问移动平台可以使用么

### DISCUSSION THREAD

**Discussion by arror** [zu1k/good-mitm#24] [Q&A] [answered]
例如编译成.a静态库给iOS使用。

**Comment by zu1k** [comment, marked as answer]
我没有测试过，也没有苹果设备来测试，理论上可行。安卓的话难度较大，得root后才能信任自己签发的证书。



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2022年3月29日(周二) 凌晨0:56
收件人: ***@***.***&gt;;
抄送: ***@***.***&gt;;
主题: [zu1k/good-mitm] 请问移动平台可以使用么 (Discussion #24)




 
例如编译成.a静态库给iOS使用。
